UNRELEASED.md

```diff
@@ -10,9 +10,12 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 - Added `flush` prop to `Card.Section` ([#3645](https://github.com/Shopify/polaris-react/pull/3645))
 - Added `stretchContent` prop for `Button` ([#3664](https://github.com/Shopify/polaris-react/pull/3664))
+- Extracted `TagsWrapper` from `Filters` for testability ([#3688](https://github.com/Shopify/polaris-react/pull/3688))
```

Comment:
@BPScott Wasn't sure where this should go. Let me know if it needs to be moved/re-worded. :)

Reply:
This feels like a "Code quality" entry - it doesn't have any effect on external users (functionality remains identical) but makes our codebase a bit nicer.